docs/troubleshooting.rst

```diff
@@ -276,3 +276,10 @@ the log level of the command to the DEBUG level and the ``-X`` option shows stac
 .. code-block:: sh
 
    $ inmanta -vvv export -X
+
+Debugging
```

Comment:
I think we should also add rpdb as an optional dependency to the setup.py? 
(like this https://godatadriven.com/blog/a-practical-guide-to-using-setup-py/#extras-require)